.github/workflows/semantic-release.yaml

```diff
@@ -7,16 +7,17 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
-    if: "!contains(github.event.head_commit.message, '[skip ci]')"
+    if: ${{ !contains(github.event.head_commit.message, '[skip ci]') }}
```

Comment:
this is no longer required. since 2021, this is automatically respected

Reply:
good to learn, we have this all over the place :)